metadata-ingestion/src/datahub/ingestion/source/unity/config.py

```diff
@@ -88,6 +88,14 @@ class UnityCatalogSourceConfig(
         description="Name of the workspace. Default to deployment name present in workspace_url",
     )
 
+    ingest_all_metastores: bool = pydantic.Field(
+        default=False,
+        description=(
+            "Ingest all metastores that match `metastore_id_pattern`, which requires the account admin role. "
```

Comment:
We should also probably raise a config error if both metastore allow/deny patterns and `ingest_only_assigned_metastore` are set

Reply:
Is there an easy way to do this when `metastore_id_pattern` has a non-null default? I guess I can check equality, just feels a bit sketch